components/org.wso2.carbon.identity.application.authenticator.oidc/src/main/java/org/wso2/carbon/identity/application/authenticator/oidc/OpenIDConnectAuthenticator.java

```diff
@@ -129,6 +148,12 @@ public boolean canHandle(HttpServletRequest request) {
         return false;
     }
 
+    private boolean isCodeResponse(HttpServletRequest request) {
```

Comment:
Code response can return an error too?
Should we handle those?

Reply:
i think here we are only checking whether code is available as a request parameter so there won't be other errors. Better to change the method name to reflect that.

Reply:
we can use something like "hasCodeParamInRequest" or something similar

Reply:
Resolved with https://github.com/wso2-extensions/identity-outbound-auth-oidc/pull/87/commits/ece7b3914049dc4fe673ce47521e484b580b3c27.